aspnetcore/blazor/file-uploads.md

```diff
@@ -867,6 +934,23 @@ Blazor relies on <xref:Microsoft.AspNetCore.SignalR.HubOptions.MaximumParallelIn
 
 Increasing the value leads to a high probability that `CopyTo` operations throw `System.InvalidOperationException: 'Reading is not allowed after reader was completed.'`. For more information, see [MaximumParallelInvocationsPerClient > 1 breaks file upload in Blazor Server mode (`dotnet/aspnetcore` #53951)](https://github.com/dotnet/aspnetcore/issues/53951).
 
+## Troubleshoot
```

Comment:
(not sure which is the more standard terminology for our docs)
```suggestion
## Troubleshooting
```

Reply:
No 🎲🎲 ... no gerunds in section headings per the docs style manual ... they don't translate well into other languages.